src/move/rewards/README.md

```diff
@@ -0,0 +1,74 @@
+<!---
+cspell:word undergassing
+cspell:word permissionlessly
+-->
+
+# `EmojicoinDotFunRewards`
+
+This package provides an overloaded swap function that randomly rewards a fixed
+percentage of swaps that exceed of an arbitrary volume. Funds are autonomously
+disbursed from a resource account with an APT vault that can be topped off
+permissionlessly.
+
+Per the [undergassing docs], the winning path requires the most gas per the
+final logical branch:
```

Comment:
```suggestion
Per the [undergassing docs], one way to mitigate undergassing attacks is to make the most optimal path cost the most gas. We've ensured that the winning path requires the most gas per the final logical branch:
```

Check my logic/wording here, but just adding a brief expositional sentence about the context for why the docs are linked and what is being referenced here

Reply:
Addressed in 7c3c6fb